ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,303 @@
+                                </q>
+                            </incipit>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="182">182</measure>
+                                    <dimensions type="outer"/>
+                                    <height unit="mm" cert="medium">250</height>
+                                    <width unit="mm" cert="medium">200</width>
+                                    
+                                </extent>
+                                <foliation>Foliation in the upper right corner of the folia.</foliation>
+                                <collation>
+                                    <note>The quire structure cannot be ascertained on the basis of the available images.</note>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="18"> 
+                                    <ab type="pricking">Pricking is partly visible.</ab>
+                                    <ab type="ruling">Ruling is not visible.</ab>
+                                    <ab type="ruling" subtype="pattern" cert="low">1A-1A-1A1A/0-0/0-0/C</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>  
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">18th-century script</seg> 
+                                <seg type="ink">Black, red.</seg>
+                                <desc>Careful, irregular; by a good scribe. <persName ref="PRS1071dAbbadi"/> describes the handwriting as <foreign xml:lang="fr">ʻguịlḥ moderneʼ</foreign>.
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#2v"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene">: <persName ref="PRS9848Walatta"/></ref>Black and white image does not allow any certain interpretation of the miniature.</desc>
```

Comment:
I remember, that we discussed the difference between miniature and drawing. If I remember right, everything in black and white is a drawing, but not a miniature.

Reply:
I think the miniatures are in colour, the available photos are black and white